recipe/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "lightweight-mmm" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ name.replace('-', '_') }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3e82b076400cd00eda80be20f23d4a1e3b6192846fbc6d9151d3155f61aeac13
+  patches:
+    - 0001-increase-mathplot-version.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  # Supported versions
+  # https://github.com/google/lightweight_mmm/blob/v0.1.9/setup.py#L74
+  skip: True # [py<310 or py>=311 or s390x]
+  
+
+requirements:
+  build: 
+    - patch # [not win]
+    - m2-patch  # [win]
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - absl-py
+    - arviz >=0.11.2
+    - immutabledict >=2.0.0
+    - jax >=0.3.18
+    - jaxlib >=0.3.18
+    # matplotlib-base ==3.6.1 in the requirements but not avaliable in conda main channel.
+    # patch 0001-increase-mathplot-version is used
+    - matplotlib-base ==3.6.2
+    - numpy >=1.23
+    - numpyro >=0.9.2
+    - pandas >=1.1.5
+    - scipy
+    - seaborn ==0.11.1
+    - scikit-learn
+    - statsmodels >=0.13.0
+    # Windows is not using latest tensorflow.
+    # https://www.tensorflow.org/install/pip#windows-native
+    # TensorFlow 2.10 was the last TensorFlow release that supported GPU on native-Windows.
+    # Linux and OSX uses 2.17.0, WIN 2.10.0
+    - tensorflow >=2.7.2
+
+test:
+  source_files:
+    - lightweight_mmm
+  imports:
+    - lightweight_mmm
+  commands:
+    - pip check # [not win]
+    # Test disabled. lightweight-mmm ticket:
+    # TypeError: where() got some positional-only arguments passed as keyword arguments: 'condition, x, y'
+    # https://github.com/google/lightweight_mmm/issues/321
+    # Tests works only with (jax==0.4.20 jaxlib==0.4.20 scipy==1.12.0)
+    # https://github.com/google/lightweight_mmm/issues/321#issuecomment-2195724411
+    #- pytest -v lightweight_mmm
```

Comment:
This is worrying. It might mean that we need to pin specific versions or add upper bounds to dependencies in order to make it work (run dependencies, e,g, `scipy <=1.12.0`).

One other thing is to add `jax ==0.4.20`, `jaxlib==0.4.20` pinnings or equivalent in the test dependencies